settings.ini

```diff
@@ -15,7 +15,7 @@ language = English
 custom_sidebar = True
 license = apache2
 status = 2
-requirements = numpy numba pandas scikit-learn quadprog matplotlib tqdm
+requirements = numpy numba pandas scikit-learn quadprog matplotlib tqdm icc_rt;platform_system!="Darwin"
```

Comment:
How/where is this used?

Reply:
This enables additional optimizations (SVML, short vector math library) for Numba for x86-64 systems, yielding significant speedup on those systems.

[See the Numba docs](https://numba.readthedocs.io/en/stable/user/performance-tips.html#intel-svml), although I see that as of recently they suggest to use a smaller package instead. I updated to use the smaller package (`intel-cmplr-lib-rt`) instead.

Reply:
I see. I'm guessing we also need to restrict on the architecture, since this won't be available for ARM CPUs and I'm not sure if installing there will try to build from source. Can you please also add `and platform_machine=="x86_64"`? You can change the way these are read in `setup.py` to use commas for example to avoid having to collapse it.